doc/ref_arch/kubernetes/chapters/chapter05.md

```diff
@@ -68,30 +94,159 @@ between tenants
 - Enable Role-Based Access Control (RBAC)
 
 ## 5.3 Node Hardening
-Ensure Kubernetes nodes are secure, hardened and configured correctly following
-well known security framework e.g. CIS benchmark, etc. Restrict administrative
-access to Kubernetes nodes while avoiding direct access to nodes for operational
-activities including debugging, troubleshooting, and other tasks.
+### 5.3.1 Node hardening: Securing Kubernetes hosts         
+When an operating system or application is installed, it comes with default
+settings. Usually, all ports are open, and all application services are turned
+on. In other words, freshly installed assets are highly insecure.
 
-## 5.4 Authentication & Authorisation
+Ensure Kubernetes nodes are secure, hardened and configured correctly following
+well known security frameworks. Security benchmarks, for example, CIS benchmarks
+are a set of configuration standards and best practices designed to help ‘harden’
+the security of their digital assets. 
+
+### 5.3.2 Restrict direct access to nodes
+Restrict root/administrative access to Kubernetes nodes while avoiding direct
+access to nodes for operational activities including debugging, troubleshooting,
+and other tasks.
+
+### 5.3.3 Vulnerability assessment
+Vulnerability assessments are a crucial part of IT risk management lifecycles.
+The mitigation of the vulnerabilities helps in protecting systems and data from unauthorized access and breaches.
+Implement necessary vulnerability scanner tools (Ex: OpenVAS and many other
+opensource and commercial tools) to identify threats and flaws within the
+infrastructure that represents potential risks.
+
+### 5.3.4 Patch management 
+Patch management is another key aspect of IT risk management lifecycle for
+security, compliance, uptime, feature enhancements and so on. Implement
+necessary patch management to ensure your environment is not susceptible to
+exploitation.
+    
+## 5.4 Securing Kubernetes orchestrator 
+
+### 5.4.1 Control network access to sensitive ports
+Kubernetes clusters usually listen on a range of well-defined and distinctive
+ports which makes it easy to identify the clusters and attack them. Hence, it is
+highly recommended to configure authentication and authorization on the cluster
+and cluster nodes.
+
+Here is an overview of the default ports used in Kubernetes. Make sure that your
```

Comment:
Can you add a reference to the source? Thanks

Reply:
Sure. Is there a format to add reference? please suggest

Reply:
@ragashreeshekar Other chapters have references but the format is as per the example below:

reference text enclosed in square brackets and the closing bracket immediately followed the reference source "[reference-text](reference link)":

[45-networking-solutions](https://github.com/cntt-n/CNTT/blob/master/doc/ref_arch/kubernetes/chapters/chapter04.md#45-networking-solutions)